### DISCUSSION TITLE
Passing Command Line Arguments?

### DISCUSSION THREAD

**Discussion by Stretch655** [zero-plusplus/vscode-autohotkey-debug#298] [Q&A]
How would I pass command line arguments (via A_Args) to the script I'm debugging?

**Comment by zero-plusplus** [comment]
The `args` attribute of [launch.json](https://code.visualstudio.com/docs/editor/debugging#_launch-configurations) can be used to pass values to A_Args.

The following is an example of a minimal launch.json using the `args` attribute.

```jsonc
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "AutoHotkey Debug",
            "type": "autohotkey",
            "request": "launch",
            "args": [
                "arg1",
                "arg2",
            ],
        }
    ]
}
```
